### PR TITLE
https://github.com/etcd-io/etcd/issues/11114

### PR DESCRIPTION
**Pull request by zhangjianweibj**
fix percent sign(%) print in log twice issue.

### CODE REVIEW

**Comment by jingyih** [comment]
Hi @zhangjianweibj, thanks for help fixing this! Could you update the commit title to `<package>: <description>`, something like `etcdserver: remove dup percentage sign in log`.

**Comment by jingyih** [comment]
`%%` is the string literal for the percent sign in fmt package [1]. It appears that zap logger is interpreting it differently.

[1] https://golang.org/pkg/fmt/#hdr-Printing

File: etcdserver/metrics.go [hunk 1]
```diff
@@ -207,9 +207,9 @@ func monitorFileDescriptor(lg *zap.Logger, done <-chan struct{}) {
 		}
 		if used >= limit/5*4 {
 			if lg != nil {
-				lg.Warn("80%% of file descriptors are used", zap.Uint64("used", used), zap.Uint64("limit", limit))
+				lg.Warn("80% of file descriptors are used", zap.Uint64("used", used), zap.Uint64("limit", limit))
 			} else {
-				plog.Warningf("80%% of the file descriptor limit is used [used = %d, limit = %d]", used, limit)
```
Comment (jingyih) [inline etcdserver/metrics.go]: %% here is probably necessary.

Comment (zhangjianweibj) [reply etcdserver/metrics.go]: ok,very thanks.

**Comment by jingyih** [comment]
Could you squash the commits into 1 commit?

**Comment by zhangjianweibj** [comment]
ok.i will push a new branch with 1 commit.

**Comment by zhangjianweibj** [comment]
new pr : https://github.com/etcd-io/etcd/pull/11118
this pull request will close.

**Pull request closed without merging**